data/ui/oops-window.ui

```diff
@@ -1,257 +1,98 @@
-      </object>
-    </child>
-  </object>
-  <object class="GtkMenu" id="menu_problem_item">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="accel_group">ag_accelerators</property>
-    <child>
-      <object class="GtkMenuItem" id="mi_report">
-        <property name="tooltip_text" translatable="yes">Submits selected problem</property>
-        <property name="related_action">gac_report</property>
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
-      </object>
-    </child>
-    <child>
-      <object class="GtkMenuItem" id="mi_delete">
-        <property name="tooltip_text" translatable="yes">Delete selected problems</property>
-        <property name="related_action">gac_delete</property>
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
-      </object>
-    </child>
-    <child>
-      <object class="GtkMenuItem" id="mi_detail">
-        <property name="tooltip_text" translatable="yes">Show technical details</property>
-        <property name="related_action">gac_detail</property>
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
-      </object>
-    </child>
-    <child>
-      <object class="GtkMenuItem" id="mi_open_directory">
-        <property name="tooltip_text" translatable="yes">Opens the selected problem's data directory in the default file system browser</property>
-        <property name="related_action">gac_open_directory</property>
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
-      </object>
-    </child>
-    <child>
-      <object class="GtkMenuItem" id="mi_copy_id">
-        <property name="tooltip_text" translatable="yes">Stores the selected problem's ID in Clibpoard</property>
-        <property name="related_action">gac_copy_id</property>
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
-      </object>
-    </child>
-    <child>
-      <object class="GtkMenuItem" id="mi_search">
-        <property name="tooltip_text" translatable="yes">Show filter box</property>
-        <property name="related_action">gac_search</property>
-        <property name="can_focus">False</property>
-      </object>
-    </child>
-  </object>
-  <object class="GtkImage" id="image1">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="icon_name">object-select-symbolic</property>
-  </object>
-  <object class="GtkImage" id="image2">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="icon_name">edit-find-symbolic</property>
-  </object>
-  <object class="GtkWindow" id="wnd_main">
+  <template class="OopsWindow" parent="GtkApplicationWindow">
```

Comment:
Maybe worth to mention explicitely in the commit message that this starts to use GtkApplicationWindow, it does a few things on the background that might be confusing for non-gtk devs (exporting actions to dbus, etc.)

Reply:
It used `GtkApplicationWindow` before this as well.

Reply:
oh colio